app/javascript/dashboard/helper/specs/conversationHelper.spec.js

```diff
@@ -0,0 +1,292 @@
+import {
+  filterDuplicateSourceMessages,
+  lastMessage,
+  readMessages,
+  unReadMessages,
+} from '../conversationHelper';
+
+const conversationItem = {
```

Comment:
@iamsivin Wrote this guide to start documenting the best practices in tests. Please take a look and rewrite this test file. 

https://chatwoot.com/hc/handbook/articles/1723432821-testing-_-best-practices#1-use-fixtures-for-reusable-test-data-and-payloads-that-are-large-enough-to-read

cc: @scmmishra

Reply:
Sure will update.

Reply:
Done